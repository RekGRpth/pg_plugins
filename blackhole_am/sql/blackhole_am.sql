CREATE EXTENSION blackhole_am;
CREATE TABLE blackhole_tab (a int) USING blackhole_am;
SELECT * FROM blackhole_tab;
INSERT INTO blackhole_tab VALUES (1);
SELECT * FROM blackhole_tab;
UPDATE blackhole_tab SET a = 0 WHERE a = 1;
SELECT * FROM blackhole_tab;
DELETE FROM blackhole_tab WHERE a = 1;
SELECT * FROM blackhole_tab;
