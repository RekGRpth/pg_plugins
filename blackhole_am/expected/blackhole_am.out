CREATE EXTENSION blackhole_am;
CREATE TABLE blackhole_tab (a int) USING blackhole_am;
SELECT * FROM blackhole_tab;
 a 
---
(0 rows)

INSERT INTO blackhole_tab VALUES (1);
SELECT * FROM blackhole_tab;
 a 
---
(0 rows)

UPDATE blackhole_tab SET a = 0 WHERE a = 1;
SELECT * FROM blackhole_tab;
 a 
---
(0 rows)

DELETE FROM blackhole_tab WHERE a = 1;
SELECT * FROM blackhole_tab;
 a 
---
(0 rows)

